### PR TITLE
fig: add head

### PR DESCRIPTION
**Pull request by ahbeng**

### CODE REVIEW

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
```
==> /usr/local/Cellar/fig/1.4.0/bin/docker-compose --version
Traceback (most recent call last):
  File "/usr/local/Cellar/fig/1.4.0/libexec/bin/docker-compose", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/pkg_resources.py", line 2797, in <module>
    parse_requirements(__requires__), Environment()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/pkg_resources.py", line 576, in resolve
    raise DistributionNotFound(req)
pkg_resources.DistributionNotFound: docker-py>=1.3.1,<1.4
Error: fig: failed
<0> expected but was
<1>.
```

You may need to tuck the `docker-py` resource in `stable` and `head` blocks respectively here.

**Comment by ahbeng** [comment]
Just did, thanks. Forgot that I'd to do a `--build-bottle` when testing `stable` locally.

File: Library/Formula/fig.rb [hunk 1]
```diff
@@ -21,9 +22,43 @@ class Fig < Formula
   depends_on "docker" => :recommended
   depends_on "docker-machine" => :recommended
 
-  resource "docker-py" do
-    url "https://pypi.python.org/packages/source/d/docker-py/docker-py-1.3.1.tar.gz"
-    sha256 "743f3fc78f6159d14ac603def6470cf1b4edefc04de8b1ad8c349b380b503f50"
+  stable do
+    resource "docker-py" do
```
Comment (DomT4) [inline Library/Formula/fig.rb]: Can you stick the URLs in their respective blocks as well, Thanks!

```ruby
stable do
  url ""
  sha256 ""

  resource "example" do
    url ""
    sha256 ""
  end
end

head do
  url "https://github.com/docker/compose.git"

  resource "example" do
    url ""
    sha256 ""
  end
end
```

**Comment by ahbeng** [comment]
Moved stable and head URLs to their respective blocks as requested.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/ad0981523dbcb309cc68fd9c3cb9ac8e05820a01. Thank you for your contribution to Homebrew @ahbeng!

**Pull request closed without merging**